### PR TITLE
feat: Zitadel translated into Spanish

### LINKED ISSUE #5585: Language: Spanish 🇪🇸
As a Spanish speaking person I want to be able to read ZITADEL login, user profile and console in Spanish

Acceptance criteria

 - [ ] The whole login is shown in Spanish if I have Spanish as my default language set
 - [ ] The user profile is shown in Spanish if I have Spanish as my default language set
 - [ ] The console is shown in Spanish if I have Spanish as my default language set
 - [ ] Emails are sent in Spanish if I have Spanish as my default language set
 
As Spanish is my mother tongue, I'd like to assign this task to me :smile:

Related to issue #4244

### PR DESCRIPTION
**Pull request by doncicuto**
Zitadel strings and messages translated into Spanish. Also some typos found in English files have been fixed.

Closes #5585 

```[tasklist]

### Definition of Ready

- [X] I am happy with the code
- [X] Short description of the feature/issue is added in the pr description
- [X] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [X] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [X] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

File: console/src/assets/i18n/es.json [hunk 1]
```diff
@@ -0,0 +1,2151 @@
...
+    "SHORTCUTS": {
+      "SHORTCUTS": "Accesos directos",
+      "SETTINGS": "Accesos directos disponibles",
+      "PROJECTS": "Proyectos",
+      "REORDER": "Mantén y arrastra el panel para moverlo",
+      "ADD": "Mantén y arrastrq el panel para añadirlo"
+    }
+  },
+  "ONBOARDING": {
+    "DESCRIPTION": "Tu proceso de embarque",
```
[the remark this answers is not in the document]
Comment (peintnermax) [reply console/src/assets/i18n/es.json:46]: found a missing translation from a recent PR..

**Review by peintnermax — CHANGES_REQUESTED** [review]
Thx for this amazing work.. Love to have spanish translations.. 🚀 console changes work and LGTM @eliobischof could you  take a look at the backend code?

File: console/src/assets/i18n/es.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (doncicuto) [reply console/src/assets/i18n/es.json:46]: Sure @peintnermax , I'll upload the missing translation today after work

[the remark this answers is not in the document]
Comment (doncicuto) [reply console/src/assets/i18n/es.json:46]: Uploaded! Did a deeper search to find more missing translations. Thanks @peintnermax and @eliobischof as always for your reviewing time

File: internal/notification/static/i18n/es.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+InitCode:
+  Title: ZITADEL - Inicializar usuario
+  PreHeader: Inicializar usuario
+  Subject: Inicializar usuario
+  Greeting: Hola {{.FirstName}} {{.LastName}},
```
Comment (livio-a) [inline internal/notification/static/i18n/es.yaml:5]:
```suggestion
  Greeting: Hola {{.DisplayName}},
```

File: internal/notification/static/i18n/es.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  Text: Este usuario fue creado en ZITADEL. Utiliza el nombre de usuario {{.PreferredLoginName}} para iniciar sesión. Por favor, haz clic en el botón más abajo para finalizar el proceso de inicialización. (Código {{.Code}}) Si no solicitaste este correo electrónico, por favor ignóralo.
+  ButtonText: Finalizar inicialización
+PasswordReset:
+  Title: ZITADEL - Restablecer contraseña
+  PreHeader: Restablecer contraseña
+  Subject: Restablecer contraseña
+  Greeting: Hola {{.FirstName}} {{.LastName}},
```
Comment (livio-a) [inline internal/notification/static/i18n/es.yaml:12]:
```suggestion
  Greeting: Hola {{.DisplayName}},
```

File: internal/notification/static/i18n/es.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  Text: Hemos recibido una solicitud de restablecimiento de contraseña. Por favor, usa el botón más abajo para restablecer tu contraseña. (Código {{.Code}}) Si no solicitaste este correo electrónico, por favor ignóralo.
+  ButtonText: Restablecer contraseña
+VerifyEmail:
+  Title: ZITADEL - Verificar correo electrónico
+  PreHeader: Verificar correo electrónico
+  Subject: Verificar correo electrónico
+  Greeting: Hola {{.FirstName}} {{.LastName}},
```
Comment (livio-a) [inline internal/notification/static/i18n/es.yaml:19]:
```suggestion
  Greeting: Hola {{.DisplayName}},
```

File: internal/notification/static/i18n/es.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  Text: Se ha añadido una nueva dirección de correo electrónico. Por favor, usa el botón más abajo para verificar tu correo electrónico. (Código {{.Code}}) Si no añadiste una nueva dirección de correo electrónico, por favor ignora este correo.
+  ButtonText: Verificar correo electrónico
+VerifyPhone:
+  Title: ZITADEL - Verificar teléfono
+  PreHeader: Verificar teléfono
+  Subject: Verificar teléfono
+  Greeting: Hola {{.FirstName}} {{.LastName}},
```
Comment (livio-a) [inline internal/notification/static/i18n/es.yaml:26]:
```suggestion
  Greeting: Hola {{.DisplayName}},
```

File: internal/notification/static/i18n/es.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  Text: Se ha añadido un nuevo número de teléfono. Por favor, usa el siguiente código para verificarlo {{.Code}}
+  ButtonText: Verificar teléfono
+DomainClaimed:
+  Title: ZITADEL - Se ha reclamado un dominio
+  PreHeader: Cambiar dirección de correo electrónico / nombre de usuario
+  Subject: Se ha reclamado un dominio
+  Greeting: Hola {{.FirstName}} {{.LastName}},
```
Comment (livio-a) [inline internal/notification/static/i18n/es.yaml:33]:
```suggestion
  Greeting: Hola {{.DisplayName}},
```

File: internal/notification/static/i18n/es.yaml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  Text: El dominio {{.Domain}} ha sido reclamado por una organización. Tu usuario actual {{.Username}} no es parte de esta organización. Por tanto tendrás que cambiar tu dirección de correo electrónico cuando inicies sesión. Hemos creado un nombre de usuario temporal ({{.TempUsername}}) para este inicio de sesión.
+  ButtonText: Iniciar sesión
+PasswordlessRegistration:
+  Title: ZITADEL - Añadir inicio de sesión sin contraseña
+  PreHeader: Añadir inicio de sesión sin contraseña
+  Subject: Añadir inicio de sesión sin contraseña
+  Greeting: Hola {{.FirstName}} {{.LastName}},
```
Comment (livio-a) [inline internal/notification/static/i18n/es.yaml:40]:
```suggestion
  Greeting: Hola {{.DisplayName}},
```

File: internal/notification/static/i18n/es.yaml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+  Text: Hemos recibido una solicitud para añadir un token para iniciar sesión sin contraseña. Por favor, usa el botón más abajo para añadir tu token o dispositivo para un inicio de sesión sin contraseña.
+  ButtonText: Añadir inicio de sesión sin contraseña
+PasswordChange:
+  Title: ZITADEL - La contraseña de usuario ha sido cambiada
+  PreHeader: Cambiar contraseña
+  Subject: La contraseña de usuario ha sido cambiada
+  Greeting: Hola {{.FirstName}} {{.LastName}},
```
Comment (livio-a) [inline internal/notification/static/i18n/es.yaml:47]:
```suggestion
  Greeting: Hola {{.DisplayName}},
```

File: internal/static/i18n/en.yaml [hunk 9]
```diff
@@ -1061,7 +1060,7 @@ EventTypes:
         changed: Label policy changed
         font:
           added: Font added to label policy
-          removed: Font remvoed from label policy
+          removed: Font remove from label policy
```
Comment (eliobischof) [inline internal/static/i18n/en.yaml:1063]:
```suggestion
          removed: Font removed from label policy
```

**Review by eliobischof — CHANGES_REQUESTED** [review]

**Review by livio-a — CHANGES_REQUESTED** [review]
Hey @doncicuto we recently changed the mail templates to use the `DisplayName` instead of `FirstName` and `LastName` to be able to make them optional later on

**Review by eliobischof — APPROVED** [review]

**Review by peintnermax — APPROVED** [review]

**Review by livio-a — APPROVED** [review]

**Pull request merged**